include/deal.II/base/symmetric_tensor.h

```diff
@@ -1217,14 +1217,14 @@ namespace internal
       case 2:
         return (data[0] * sdata[0] +
                 data[1] * sdata[1] +
-                Number(2.) * data[2] * sdata[2]);
```

Comment:
@davydden but `std::complex` should define multiplication of complex numbers with real scalars, shouldn't it? If you look at the change in 8adaf35 you will realize that the PR includes a random mix of going from `2 * x` to `Number(2.) * x` or `2. * x`. Since that patch does alright with `2. * x` in some occasions (see e.g. line 1264, https://github.com/dealii/dealii/blob/master/include/deal.II/base/symmetric_tensor.h#L1264), I suggest you grep for all `Number(2.) *` and replace it by `2. *`. It looks like #2792 introduced tests for this case, so it is definitely necessary to check `ctest -R symmetric_tensor` tests.